build.yaml

```diff
@@ -160,10 +163,20 @@ steps:
  - kind: buildImage
    name: scorecard_image
    dockerFile: scorecard/Dockerfile
-   contextPath: scorecard
+   contextPath: .
```

Comment:
What is contextPath?

Reply:
It's path where the docker build command is run.

Reply:
So this depends on whether the module depends on anything held outside of its root (for instance, something that depends on /auth) since docker cannot copy anything outside of its context. Correct?

Reply:
Correct.  I'm not sure how careful we are about always having the lowest possible context path.  `scorecard` might work here again: before I was installing gear in this image, but now I'm putting it in the services base image.

Reply:
>  You mean the "Hail" in the center the / page?

Yes

Reply:
I left it because I'm going to turn the / page into a dashboard that has a summary box for each service: link to a running notebook, activate and recent batch jobs, PR status for developers, etc.

Reply:
Right but why get rid of all <hN> styling? You have a bunch of <hN> elements.

Reply:
Right, but why remove all styling on <hN> elements?

> Oops, sorry, I misread where the failure was happening. You need `cseed/hail:batch-web`, not `cseed:batch-web`. I created an issue to fix this: #7074

Not entirely working, but I'm happy to just review what you have in your namespace.

For reference:
```sh
received unexpected HTTP status: 503 Service Unavailable
```

https://ci.hail.is/batches/13547

Reply:
> Right, but why remove all styling on elements?

It was just styling for headers. Mainly, I didn't like how the styling you had looked with how I was using headers in the HTML.

I'm slightly confused why you're using a header tag on what you're calling a logo.  I thought headers were mainly used to tag/name sections of a page.

Reply:
Sorry, "heading"

Reply:
> received unexpected HTTP status: 503 Service Unavailable

It looks like this is from GCR while doing a docker push.  Surprising, I've never seen it  before.  Is it reproducible?  I bet GCR just had a temporary hiccup.